docs/server/source/production-deployment-template-tendermint/workflow.rst

```diff
@@ -0,0 +1,188 @@
+
+Use the self-signed CA to sign those three CSRs:
+
+* Three certificates (one for each CSR).
+
+For help, see the pages:
+
+* :doc:`How to Generate a Server Certificate for MongoDB <../production-deployment-template/server-tls-certificate>`
+* :doc:`How to Generate a Client Certificate for MongoDB <../production-deployment-template/client-tls-certificate>`
+
+☐ Make up an FQDN for your BigchainDB node (e.g. ``mynode.mycorp.com``).
+Make sure you've registered the associated domain name (e.g. ``mycorp.com``),
+and have an SSL certificate for the FQDN.
+(You can get an SSL certificate from any SSL certificate provider.)
+
+☐ Ask the managing organization for the user name to use for authenticating to
+MongoDB.
+
+☐ If the cluster uses 3scale for API authentication, monitoring and billing,
+you must ask the managing organization for all relevant 3scale credentials -
+secret token, service ID, version header and API service token.
+
+☐ If the cluster uses MongoDB Cloud Manager for monitoring,
+you must ask the managing organization for the ``Project ID`` and the
+``Agent API Key``.
+(Each Cloud Manager "Project" has its own ``Project ID``. A ``Project ID`` can
+contain a number of ``Agent API Key`` s. It can be found under
+**Settings**. It was recently added to the Cloud Manager to
+allow easier periodic rotation of the ``Agent API Key`` with a constant
+``Project ID``)
+
+
+.. _generate-the-blockchain-id-and-genesis-time:
+
+3. Generate the Blockchain ID and Genesis Time
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tendermint nodes require two parameters that need to be common and shared between all the
+participants in the network.
+
+* ``chain_id`` : ID of the blockchain. This must be unique for every blockchain.
+
+  * Example: ``0001-01-01T00:00:00Z``
+
+* ``genesis_time`` : Official time of blockchain start.
+
+  * Example: ``test-chain-9gHylg``
+
+The following parameters can be generated using the ``tendermint init`` command.
+To `initializae <https://tendermint.readthedocs.io/en/master/using-tendermint.html#initialize>`_.
+You will need to `install Tendermint <https://tendermint.readthedocs.io/en/master/install.html>`_
+and verify that a ``genesis.json`` file in created under the `Root Directory
+<https://tendermint.readthedocs.io/en/master/using-tendermint.html#directory-root>`_. You can use
+the ``genesis_time`` and ``chain_id`` from this ``genesis.json``.
+
+Sample ``genesis.json``:
+
+.. code:: json
+
+    {
+    "genesis_time": "0001-01-01T00:00:00Z",
+    "chain_id": "test-chain-9gHylg",
+    "validators": [
+        {
+        "pub_key": {
+            "type": "ed25519",
+            "data": "D12279E746D3724329E5DE33A5AC44D5910623AA6FB8CDDC63617C959383A468"
+        },
+        "power": 10,
+        "name": ""
+        }
+    ],
+    "app_hash": ""
+    }
+
+
+
+☐ :doc:`Deploy a Kubernetes cluster on Azure <../production-deployment-template/template-kubernetes-azure>`.
```

Comment:
I think this links to the wrong page. Shouldn't it link to the BigchainDB + Tendermint one?